test/integration/alertsMenuView_integrationTests.js

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /*global sinon, todo, chai, describe, it, calledOnce*/
 //integration tests
-
 describe('alertsMenuView.js spec', function() {
+
+    var now_date, old_date, alert_recent, alert_old;
```

Comment:
these can be declared inside the beforeEach block as:
```
this.now_date = ...
this.old_date = ...
```
and then when you refer to the variables in the tests below you will have access to the `this` context.

example:
```
// if change, then highlight
this.testCollection.reset();
this.testCollection.add({
  results: [this.alert_old]
});
// change with old alert, then no highlight
this.testView.update();
...
```